### PR TITLE
docs: add note about `cypto.randomUUID()`

### LINKED ISSUE #682: Start recommending Web Crypto API for RFC version 4?
Now that [`Crypto.randomUUID()`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) is supported in all browsers and Node.js it might be good to point out to users that they might not need a library. Perhaps this could be documented in the README?

### PR DESCRIPTION
**Pull request by jonkoops**
Closes #682

### CODE REVIEW

File: README_js.md [hunk 1]
```diff
@@ -32,7 +32,11 @@ For the creation of [RFC4122](https://www.ietf.org/rfc/rfc4122.txt) UUIDs
 - **Small** - Zero-dependency, small footprint, plays nice with "tree shaking" packagers
 - **CLI** - Includes the [`uuid` command line](#command-line) utility
 
-**Upgrading from `uuid@3`?** Your code is probably okay, but check out [Upgrading From `uuid@3`](#upgrading-from-uuid3) for details.
+> **Note**
```
Comment (jonkoops) [inline README_js.md:35]: This applies some special GitHub flavored markdown to visualize the notes:

> **Note**
> Upgrading from `uuid@3`? Your code is probably okay, but check out [Upgrading From `uuid@3`](#upgrading-from-uuid3) for details.

> **Note**
> Only interested in creating a version 4 UUID? You might be able to use [`cypto.randomUUID()`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID), eliminating the need to install this library.

Comment (LinusU) [reply README_js.md:35]: Cool! I didn't know about this

**Review by LinusU — COMMENTED** [review]
Changes looks good 👍 

Could you run `npm run docs` so that the rendered readme is also updated?

**Comment by jonkoops** [comment]
@LinusU Done 👍

**Review by LinusU — APPROVED** [review]
LGTM 👍

**Comment by broofa** [comment]
@jonkoops Thanks for the help!

<image width="300" src="https://media.giphy.com/media/hmVVRM1uV7vYA/giphy.gif" />

**Pull request merged**